protocols/p10.py

```diff
@@ -12,6 +13,8 @@
 from pylinkirc.log import log
 from pylinkirc.protocols.ircs2s_common import *
 
+__all__ = ['P10Protocol']
```

Comment:
Since these are single class modules I wouldn't be opposed to just importing the protocol name explicitly from dependants and skipping this.